src/scss/components/_list.scss

```diff
@@ -100,4 +109,97 @@
       padding-right: inherit;
     }
   }
+
+  &__title {
+    color: get-color(rastaGray);
+    display: block;
+    font-size: 12px;
+    font-weight: get-weight(bold);
+    text-align: left;
+    text-transform: uppercase;
+  }
+
+  &--on-column {
+    &[class*='col-'] {
+      display: inline-block;
+      margin-left: 0;
+
+      + & {
+        margin-left: auto;
+        margin-right: 0;
+      }
+
+      .list__item {
+        width: 100%;
+      }
+    }
+  }
+
+  &--social {
+    display: flex;
+    flex-grow: 1;
+
+    .list__item {
+      border-radius: 50%;
+      height: 48px;
+      margin-left: 6%;
+      text-indent: -9999px;
+      width: 48px;
+
+      a {
+        padding: 0;
+        width: 100%;
+      }
+
+      img {
+        display: block;
+        width: 100%;
+      }
+
+      &:first-child {
+        margin-left: 0;
+      }
+    }
+  }
+
+  &--app {
```

Comment:
This is the same case of `list--social`. This is defining what the modifier is, not what modifies. My suggestion is to use another modifier, to make `flex-direction: row` with icons.

Visually, the `list--social` and `list--app` are different just on the direction of columns. Why can not we use the same `list--with-icons` and another modifier to change direction of columns? If is hard to have just a modifier `list--in-rows`, we can use a `list--with-row-icons` and specify what we need

Reply:
The modifier has chages in inner style modifications to `.list__item` and `.list__image`. It's more  easy to have both class modifiers: `.list--with-icons` and `.list--with-row-icons`.